README.md

```diff
@@ -89,8 +89,8 @@ Install and use by directly including the [browser files](dist).
     <meta charset="utf-8">
     <title>A-Frame Particle System Component Example</title>
     <meta name="description" content="Hello, World!">
-    <script src="https://aframe.io/releases/1.4.1/aframe.min.js"></script>
-    <script src="https://cdn.jsdelivr.net/gh/c-frame/aframe-particle-system-component@1.1.4/dist/aframe-particle-system-component.min.js"></script>
+    <script src="https://aframe.io/releases/1.6.0/aframe.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/gh/c-frame/aframe-particle-system-component@1.1/dist/aframe-particle-system-component.min.js"></script>
```

Comment:
1.1.4 was correct here

Reply:
Next version would be 1.2.0 actually, because the addUpdateRange change was introduced in r159 so it will break on aframe 1.5.0 / three 158, so we should do a minor release and not a patch release.

Reply:
I added a note about compatibility in the release notes https://github.com/c-frame/aframe-particle-system-component/releases/tag/1.2.0